go.mod

```diff
@@ -14,7 +14,6 @@ require (
 	github.com/bugsnag/osext v0.0.0-20130617224835-0dd3f918b21b // indirect
 	github.com/bugsnag/panicwrap v0.0.0-20151223152923-e2c28503fcd0 // indirect
 	github.com/denverdino/aliyungo v0.0.0-20190125010748-a747050bb1ba
-	github.com/dgrijalva/jwt-go v0.0.0-20170104182250-a601269ab70c // indirect
```

Comment:
Can you add a `// indirect` for the jwt-go module, to force it to the most current (v3.2.4) version (see my other comment)

```go
github.com/form3tech-oss/jwt-go v3.2.4+incompatible // indirect
```

Reply:
Sadly it's not that easy, v3.2.4 introduced an empty mod file which means the import line has to change to github.com/form3tech-oss/jwt-go/v2 now, which is a code change in github.com/Azure/go-autorest

Reply:
Actually I think there is no valid way to import 3.2.4 with that mod file in there.

Reply:
Perhaps someone should open a PR to remove it; looks like it may have been unintentional that it was added.

Either that and/or forward-port the security fix to the upstream (https://github.com/form3tech-oss/jwt-go/pull/14)

Reply:
I think in the main branch, cause form3tech-oss/jwt-go@v3.2.4 contains an invalid go.mod file (the module string should be versioned). Our only options are:
* Use a replace to swap the module for golang-jwt/jwt, I don't like replace directives in releases.
* Get Azure/go-autorest to change to golang-jwt/jwt (which they are doing https://github.com/Azure/go-autorest/issues/642) then update that here.

Reply:
As a workaround, it's possible to use a commit as version; in that case I think `go mod` should work (run go mod tidy afterwards to generate the pseudo version)

```go
github.com/form3tech-oss/jwt-go a211650c6ae1cff6d7347d3e24070d65dcfb1122 // indirect
```

Reply:
Also opened https://github.com/form3tech-oss/jwt-go/pull/17 to remove the go.mod 😅